### PR TITLE
Prost

### PR DESCRIPTION
**Pull request by dmadisetti**
https://github.com/AnotherGroupChat/maeve/issues/41

### CODE REVIEW

File: games/game_design.pbtxt [hunk 1]
```diff
@@ -16,33 +16,58 @@ stats {
...
+    actions {
+      verb: "OPEN"
+      conditional {
+        clause {
+          scope {
+            id: "WINDOW"
+            state: "CLOSE"
+          }
+        }
+        left_change {
```
Comment (pbateman828) [inline games/game_design.pbtxt:64]: right_change?

File: games/game_design.pbtxt [hunk 2]
```diff
@@ -16,33 +16,58 @@ stats {
...
+    description: "You are behind the white house.  In one corner of the house there is a small window which is "
+    conditional {
+      clause {
+        scope {
+          id: "WINDOW"
+          state: "OPEN"
+        }
+      }
+      left_leaf: "open."
+      right_leaf: "slightly ajar."
```
Comment (pbateman828) [inline games/game_design.pbtxt:31]: close or slightly ajar?

File: games/game_design.pbtxt [hunk 3]
```diff
@@ -16,33 +16,58 @@ stats {
...
+    actions {
+      verb: "CLOSE"
+      conditional {
+        clause {
+          scope {
+            id: "WINDOW"
+            state: "OPEN"
+          }
+        }
+        left_change {
```
Comment (pbateman828) [inline games/game_design.pbtxt:46]: ?

File: games/game_design.pbtxt (hunk 2 above)
Comment (dmadisetti) [reply games/game_design.pbtxt:31]: Slightly ajar as per the original zork manuscript

File: games/game_design.pbtxt (hunk 1 above)
Comment (dmadisetti) [reply games/game_design.pbtxt:64]: Fixed this

**Comment by pbateman828** [comment]
I think you're good.

**Pull request merged**